alignak/acknowledge.py

```diff
@@ -83,7 +85,7 @@ class Acknowledge(AlignakObject):  # pylint: disable=R0903
     # of the Alignak process. If not, the comment will be deleted the
     # next time Alignak restarts.
 
-    def serialize(self):
+    def serialize(self, filtered_fields=None):
```

Comment:
After re-reading this code, the comments need to be refactored and even the documentation of the return parameter is not correct! We will never get a json representation with such a function but a dict :wink: